spec/acceptance/peadm_spec/plans/test_migration.pp

```diff
@@ -30,6 +32,8 @@
     new_primary_host => $new_primary_target,
     upgrade_version => $upgrade_version,
     replica_host => $new_replica_target,
+    primary_postgresql_host => $primary_postgresql_target,
```

Comment:
we dont need to pass primary_postgresql_target in here , we just need to pass new_primary_postgresql_host